web/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -39,11 +39,11 @@ type PropsType = {
   languageCode: string
 }
 
-const CityNotCooperatingFooter = ({ languageCode }: PropsType): ReactElement => {
+const CityNotCooperatingFooter = ({ languageCode }: PropsType): ReactElement | null => {
   const { t } = useTranslation('landing')
 
   if (!buildConfig().featureFlags.cityNotCooperating) {
-    return <></>
+    return null
```

Comment:
Why do you prefer "null" over "<>"?

Reply:
Changed it because of this rule: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md
It is now a part of one of our extended eslint configs. I think we should stick to those rules most of the time if there is no important reason not to do so. I also think it kind of makes sense to have this consistent and to show that nothing is rendered.

Reply:
@SteffiStuffel if you have no objections I'd say `null` is the way to go here.

Reply:
Just wanted to know.